hadoop-hdfs-project/hadoop-hdfs/src/main/proto/StorageContainerDatanodeProtocol.proto

```diff
@@ -58,44 +66,61 @@ message SCMVersionRequestProto {
 * added on the fly and protocol to remain stable.
 */
 message SCMVersionResponseProto {
-  required uint32 softwareVersion = 1;
-  repeated  hadoop.hdfs.ozone.KeyValue keys = 2;
+  required uint32 softwareVersion = 2;
```

Comment:
why do we have to change the SCMVersionResponseProto.softwareVersion starting with a value of 2?